tracing/opentelemetry/src/main/java/io/helidon/tracing/opentelemetry/OpenTelemetryTracerProvider.java

```diff
@@ -112,4 +113,51 @@ public Optional<Span> currentSpan() {
     public boolean available() {
         return GLOBAL_SET.get();
     }
+
+
+    /**
+     * Check if OpenTelemetry is present by indirect properties.
+     * This class does best explicit check if "otel.agent.present" config property is set and uses its
+     * value to set the behaviour of OpenTelemetry producer.
+     *
+     * If the value is not explicitly set, the detector does best effort to estimate indirect means if the agent is present.
+     * This detector may stop working if OTEL changes the indirect indicators.
+     */
+    public static class AgentDetector {
```

Comment:
This whole class is not used anywhere. It should be removed. Not to mention, it is basically a duplication of AgentDetector from HelidonOpenTelemetry.

Reply:
Removed